### ISSUE TITLE
Onedrive : Unable to Uninstall

### ISSUE THREAD

**Issue by Rwfrk** [10se1ucgo/DisableWinTracking#364]
**INFORMATION:**

* Windows Edition (Home/Pro/Enterprise):
* Architecture (32bit/64bit):
* Options used:
W10 Dell image out of the box -> 
Previously: "Services, Clear DiagTrack log, Telemetry, Block Tracking Domains,Block tracking IP addresses, Wifisense,Uninstall Onedrive" Service Method Disable:. Mode Privacy. Go.

Onedrive failed to uninstall.Rebooted. Attempted again with just Uninstall Onedrive because everything else seemed successful. Failed.

**LOG:**

```python
13:42:28 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
13:42:28 INFO: ('Windows', 'DESKTOP-UEGHRT1', '10', '10.0.15063', 'AMD64', 'Intel64 Family 6 Model 158 Stepping 9, GenuineIntel')
13:42:28 INFO: DisableWinTracking version 3.1.2
13:42:51 INFO: IP Blocker: The IP 134.170.30.202 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 137.116.81.24 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 157.56.106.189 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 2.22.61.43 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 2.22.61.66 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 204.79.197.200 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 23.218.212.69 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 64.4.54.254 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 65.39.117.230 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 65.52.108.33 was successfully blocked.
13:42:51 INFO: IP Blocker: The IP 65.55.108.23 was successfully blocked.
13:42:51 INFO: DiagTrack: Take Ownership of AutoLogger-Diagtrack-Listener.etl was successful
13:42:51 ERROR: 1
None
13:42:51 INFO: DiagTrack: Grant Admin Privilege of AutoLogger-Diagtrack-Listener.etl was successful
13:42:51 ERROR: 2
None
13:42:51 INFO: DiagTrack: Cleared AutoLogger-Diagtrack-Listener.etl
13:42:51 INFO: DiagTrack: Deny System Privilege of AutoLogger-Diagtrack-Listener.etl was successful
13:42:51 INFO: DiagTrack: Successfully cleared and locked DiagTrack log.
13:42:51 INFO: Services: Succesfully removed service 'dmwappushsvc'
13:42:51 INFO: Services: Succesfully removed service 'DiagTrack'
13:42:51 INFO: Registry: Successfully modified dmwappushsvc key.
13:42:51 INFO: Registry: Successfully modified DiagTrack key.
13:42:51 INFO: Registry: Successfully modified AllowTelemetry key.
13:42:51 INFO: Registry: Successfully modified WifiSense Open-ness key.
13:42:51 INFO: Registry: Successfully modified WifiSense Credential Share key.
13:42:51 INFO: Registry: Successfully modified ListPin key.
13:42:51 INFO: Registry: Successfully modified FileSync key.
13:42:51 INFO: OneDrive: unable to uninstall
13:42:51 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
13:42:51 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions
13:47:46 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
13:47:46 INFO: ('Windows', 'DESKTOP-UEGHRT1', '10', '10.0.15063', 'AMD64', 'Intel64 Family 6 Model 158 Stepping 9, GenuineIntel')
13:47:46 INFO: DisableWinTracking version 3.1.2
13:47:50 INFO: Registry: Successfully modified ListPin key.
13:47:50 INFO: Registry: Successfully modified FileSync key.
13:47:50 INFO: OneDrive: unable to uninstall
13:47:50 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
13:47:50 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions
```

**Comment by Ruined1** [comment]
You can get the "unable to uninstall" message if OneDrive has already been uninstalled. Show me where OneDrive has not be uninstalled from this system.

Diagtrack issue was resolved in latest version 3.1.3

**Comment by Ruined1** [comment]
Please try the [latest release](https://github.com/10se1ucgo/DisableWinTracking/releases/latest) and the issue should be resolved. If you still need help, please create a new issue.
